src/reporter.js

```diff
@@ -9,8 +9,9 @@ const { LOG_FILES, HOME_DIR, DESIRED_BROWSER } = require('./constants');
 
 const log = logger('reporter');
 
-const region = process.env.SAUCE_REGION || 'us-west-1';
+const region = process.env.SAUCE_REGION || '';
```

Comment:
@dpgraham why removing the default?

Reply:
That was probably a mistake made in merging from origin. I'll add it back.